resources/js/store/store.js

```diff
@@ -19,6 +19,7 @@ export default new Vuex.Store({
         taskLists: {},
 
         //Character
+        experienceTable: null,
```

Comment:
Denk dat het een mooi doel voor volgende week is om de store ook op te delen

https://vuex.vuejs.org/guide/modules.html

hier staat een goed voorbeeld!